content/blog/2019/11/2019-11-21-welcome-to-the-matrix.adoc

```diff
@@ -0,0 +1,389 @@
+---
+layout: post
+title: "Welcome to the Matrix"
+tags:
+- pipeline
+- plugins
+- declarative
+- matrix 
+author: lnewman
+---
+
+I often find myself needing to run the same actions on a bunch of different configurations.  
+Up to now, that meant I had to make multiple copies of the same stages in my pipelines. 
+When I needed to make changes, I had to make the same changes in multple places throughout my pipepine.
+Maintaining even a small number of configuration was difficult for larger pipelines.
+
+Declarative Pipeline 1.5.0-beta1 (now available from the
+link:https://updates.jenkins.io/experimental/[Jenkins Expermental Update site]) adds a new `matrix` section that lets me specify a list stages once and then run that same list in parallel on multiple configurations. Let's take a look!
```

Comment:
typo  Experimental  in [Jenkins Expermental Update site]

Reply:
```suggestion
link:https://updates.jenkins.io/experimental/[Jenkins Experimental Update site]) adds a new `matrix` section that lets me specify a list stages once and then run that same list in parallel on multiple configurations. Let's take a look!
```